### PR TITLE
Adding documentation

### PR DESCRIPTION
**Pull request by asyedham**

### CODE REVIEW

**Comment by smalleni** [comment]
@venkataanil Please review this.

File: docs/source/playbooks.rst [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+------------------------------------------------------------------------------------------------------------------------------
+
+Adds undercloud to inventory in ansible controller node.
+
+`prepare_nic_configs.yml <https://github.com/redhat-performance/jetpack/blob/master/prepare_nic_configs.yml>`_
+--------------------------------------------------------------------------------------------------------------
+
+This playbook will create nic-configs for the overcloud. In `group_vars/all.yml <https://github.com/redhat-performance/jetpack/blob/master/group_vars/all.yml>`_  we define interface based on the machine type.
+
+For example, if undercloud hostname is f05-h11-000-1029p.rdu2.scalelab.redhat.com, then machine type is “1029p” and from group_vars/all.yaml
```
Comment (venkataanil) [inline docs/source/playbooks.rst]: Please remove any lab specific details, so please remove line 50.

File: docs/source/contribution.rst [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+To contribute to JetPack the skills required are:
+
+  * Familiar with Ansible
+  * Good understanding and debugging of `infrared <https://github.com/redhat-openstack/infrared>`_  ansible tasks
+  * Willigness to debug the undercloud and overcloud deployment failures :)
+
+References
+----------
+
+https://docs.google.com/document/d/1qtmjUNoKvKvVFA0QYoK7OD1tC_Q5L-qkjMoSsNSrtWI/edit?usp=sharing
```
Comment (venkataanil) [inline docs/source/contribution.rst]: Remove references section. These are our internal docs and shouldn't be shared for others.

**Review by venkataanil — COMMENTED** [review]
Awesome work Asma. If you work on my two comments, we can merge this patch.

File: docs/source/contribution.rst (hunk 2 above)
Comment (asyedham) [reply docs/source/contribution.rst]: @venkataanil Have addressed your comments. Please have a look!

**Review by venkataanil — APPROVED** [review]

**Comment by venkataanil** [comment]
Thanks for the PR.

**Pull request merged**